---
title: Genomic analysis of COVID-19 spread. Situation report 2020-04-03.
authors:
  - Emma Hodcroft
  - Cassia Wagner
  - Sidney M. Bell
  - Nicola Müller
  - James Hadfield
  - Misja Ilcisin  
  - Richard Neher
  - Trevor Bedford
authorLinks:
  - https://neherlab.org/emma-hodcroft.html
  - https://bedford.io/team/cassia-wagner/
  - https://twitter.com/sidneymbell
  - https://bedford.io/team/nicola-mueller/
  - https://bedford.io/team/james-hadfield/
  - https://bedford.io/team/misja-ilcisin/
  - https://neherlab.org/richard-neher.html
  - https://bedford.io/team/trevor-bedford/
affiliations: "Fred Hutch, Seattle, USA; Biozentrum, Basel, Switzerland; CZI, CA, USA"
translators:
translatorLinks:
date: "2020 April 03"
dataset: "https://nextstrain.org/ncov/2020-04-02?d=map&legend=closed"
abstract: "This report uses publicly shared genomic data to track the spread of COVID-19. These reports are updated weekly."
---
<!-- Translators: Only text after : in the above ^ needs to be translated -->
<!-- Comment tags like these do not need to be translated, they are only to help you! -->
<!-- Ensure that links always end in a 'letter' (. counts) If some kind of text doesn't follow them, it breaks the slide. -->
<!-- numbers can be tagged ilke this: 161</tag> - this is just for us to help find them to update! Just leave in the </tag> bit. -->

<!-- This is left-side text 1-->
# [Table of Contents](https://nextstrain.org/ncov/2020-04-02?d=tree,map&p=grid)

* [Background resources](https://nextstrain.org/narratives/ncov/sit-rep/2020-04-02?n=2).     
* [A note on sampling](https://nextstrain.org/narratives/ncov/sit-rep/2020-04-02?n=3).
* [Updates for Africa](https://nextstrain.org/narratives/ncov/sit-rep/2020-04-02?n=4).
* [Updates for Asia](https://nextstrain.org/narratives/ncov/sit-rep/2020-04-02?n=7).
* [Updates for Europe](https://nextstrain.org/narratives/ncov/sit-rep/2020-04-02?n=8).
* [Updates for North America](https://nextstrain.org/narratives/ncov/sit-rep/2020-04-02?n=11).
* [What you can do](https://nextstrain.org/narratives/ncov/sit-rep/2020-04-02?n=13).
* [Scientific credit](https://nextstrain.org/narratives/ncov/sit-rep/2020-04-02?n=14).

<!-- This is right-side text -->
```auspiceMainDisplayMarkdown
# Executive summary

We analyzed 2,544 publicly shared COVID-19 genomes. By comparing these viral genomes to each other, we can characterize how COVID-19 is moving around the world and spreading within communities. In this week's updates, we report:

**Africa**  
* Evidence for at least two different transmission chains circulating locally in Senegal  
* Continued local spread in the Democratic Republic of the Congo  

/* **Asia**  
* The origins of the Japanese outbreak may be associated with cruiseship travel */

**Europe**  
* Extensive sequencing by Iceland highlights extent of travel-associated spread
* Travel histories reveal local transmission in Austria

**North America**   
* Extensive mixing across the U.S.  
* Two local transmission chains circulating in Ontario  

**Other areas of the world**
We also received new viral genomes from South America and Oceania. Most of these new samples are scattered across the tree and group with samples collected in other regions of the world. Without more contextual data, however, we aren't yet able to draw inferences about these new sequences.
<br><br>
We applaud the groups working to generate additional data for these regions. As more specific inferences become possible, we will immediately share those findings.
```


<!-- ############ SLIDE BREAK ############# -->

<!-- This is left-side text 2-->
# [COVID-19 Resources](https://nextstrain.org/ncov/2020-03-27)
We've prepared some resources that are worth reading to familiarize yourself with COVID-19 and the virus that causes it, SARS-CoV-2, also known as hCov-19.
This information will make interpreting the data we present in this narrative easier.
<br><br>
If you aren't familiar with phylogenetic trees, we encourage you to check out the ['How to Read Phylogenies' narrative](https://nextstrain.org/narratives/trees-background/) and come back when you're ready.
<!-- This is right-side text -->
```auspiceMainDisplayMarkdown

## Nextstrain Resources  

[How to read a phylogeny](https://nextstrain.org/narratives/trees-background/).  

[Background on coronaviruses](https://nextstrain.org/help/coronavirus/human-CoV).

[Background on the current hCoV-19 outbreak](https://nextstrain.org/help/coronavirus/SARS-CoV-2).

[FAQs](https://nextstrain.org/help/coronavirus/FAQ).

[Common misconceptions](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-13?n=11).


## External Resources  
[WHO Situation Reports](https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports).

[CDC Resources](https://www.cdc.gov/coronavirus/2019-ncov/index.html).

[The NYTimes COVID-19 coverage](https://www.nytimes.com/news-event/coronavirus).

[Our World in Data](https://ourworldindata.org/coronavirus).

[Ask a Scientist & FAQs](https://covid19.fas.org/l/en).


```

<!-- ############ SLIDE BREAK ############# -->

<!-- This is left-side text 3-->
# [A note on sampling](https://nextstrain.org/ncov/2020-04-02?c=country&r=country&d=map&p=grid&legend=closed)
We currently have sequences from samples taken in 53 countries across 6 continents. This is an incredible achievement -- sequencing an unknown, large RNA virus in the midst of a pandemic is difficult, and is only possible through the incredible work and timely sharing of data by scientists and physicians around the world.
<br><br>
While this data enables us to infer many useful characteristics of the outbreak and track its spread in real time, it's important to emphasize that our conclusions are limited by the available data.
<br><br>
For example, the map shows relatively few sequences from the global south. This is NOT because COVID-19 isn't circulating in these areas, or that these cases are not as crucial to understand; rather, we just don't have much data available from these areas. The size of each circle on the map indicates how much data is currently available from that area, rather than the true size of the outbreak.

<!-- There is NO right-side text -->



<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text 4-->
# [Updates for Africa](https://nextstrain.org/ncov/2020-04-02?d=tree,map&f_region=Africa&p=grid)
<!-- There is no right side text -->


<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text 5-->
# [Two local transmission chains circulating in Senegal](https://nextstrain.org/ncov/2020-04-02?c=division&d=tree,map&f_country=Senegal&f_region=Africa&label=clade:A2a&p=grid&r=division)

Here, we see two clusters of cases from Senegal. Each of these two clusters is more closely related to samples from other parts of the world than to each other, which indicates that they are the result of 2 independent introductions.
<br><br>
The cluster shown towards the top of the tree consists of closely related cases, all sampled in Dakar between February 28 -- March 4.
This suggests that hCoV-19 has been circulating locally in Dakar for about 1 month.
<br><br>
In the middle of the tree is a cluster of cases sampled in Touba between March 10 -- March 12.
This suggests that hCoV-19 may have been circulating in Touba for around 3 weeks.
<!-- There is no right side text -->

<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text 6-->
# [Continued local transmission in the DRC](https://nextstrain.org/ncov/2020-04-02?c=division&d=tree,map&f_country=Congo&f_recency=3-7%20days%20ago&label=clade:A2a&p=grid&r=division)
We received 8 new genomes this week from Kinsasha in the Democratic Republic of the Congo.
<br><br>
All but 1 of the new samples cluster within the local outbreak clades which [we reported on last week](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=16).
This demonstrates that there is ongoing local transmission within the DRC, and swift action must be taken to contain the outbreak.


<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text 7-->
# [Updates for Asia](https://nextstrain.org/ncov/2020-04-02?d=tree,map&f_region=Asia&p=grid)

<!-- There is no right side text -->


<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text -->
<!-- # [Japanese outbreak may be cruiseship-associated]


Japanese cluster seems to have been caused from exposure to US/European circulating variants while on holiday in Egypt, which were then introduced to Japan.
Evidence: 16 sequences sampled in Japan are from Nile River cruise ship. They form 2 main clusters (unfortunately the random sorting in div view makes this a little hard to capture), but both are among primarily USA/Europe sequences.
Observation: It seems the Nile cruise ship had 2 separate variatns circulating. It seems likely that these chains came from travellers who were infected with strains circulating in the US and Europe these samples are dated 9-20 March as sampling, so cruise presumably took place earlier than that. Essentially, spread was wide enough at beginning of march that variants that primarily circulated in Europe/USA were exported ‘back’ to Asia. -->



<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text 8-->
# [Updates for Europe](https://nextstrain.org/ncov/2020-04-02?d=tree,map&f_region=Europe&p=grid)

<!-- There is no right side text -->


<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text 9-->
# [Extensive sequencing by Iceland highlights extent of travel-associated spread](https://nextstrain.org/ncov/2020-04-02?c=country_exposure&f_country=Iceland&p=grid)

We recently received 301 sequences -- from 25\% of [confirmed cases]((https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200402-sitrep-73-covid-19.pdf)) -- from Iceland!
<br><br>
Coloring each sample by its recorded travel history, we see that approximately half of these infections were acquired outside of Iceland, while half were (presumably) acquired locally.
<br><br>
As we reported last week, this highlights just how much mixing there is between the outbreaks in each European country.
For any given sample, we can draw inferences about where its predecessors must have been, but we can't rule out the possibility of indirect transmission by way of an unsampled location.
<!-- There is no right side text -->


<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text 10-->
# [Travel histories reveal local transmission in Austria](https://nextstrain.org/ncov/2020-04-02c=country_exposure&d=tree,map&f_country=Iceland&f_country_exposure=Austria&m=div&p=full)

This is a powerful example of how comprehensive sequencing efforts and travel logging by one country can help us understand the local situation elsewhere, even for places that aren't yet generating sequencing data.
<br><br>
Looking at the cases sequenced in Iceland after reported travel to Austria, we see multiple clusters of closely related Austrian cases, sequenced over a short time period.
This is most consistent with a scenario wherein there were multiple introductions into Austria, followed by community transmission.





<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text 11-->
# [Updates for North America](https://nextstrain.org/ncov/2020-04-02?d=tree,map&f_region=North%20America&p=grid)

<!-- There is no right side text -->


<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text 12-->
# [Extensive mixing across the U.S.](https://nextstrain.org/ncov/2020-04-02?c=division&f_region=North%20America&m=div&r=location)

As we gain a fuller picture of the outbreak in the U.S., it's clear that these localized outbreaks are the result of extensive mixing.
<br><br>
While many of the early cases were travel-associated, more recent cases from many different states are intermingled with each other.   
<br><br>
As we've reported in previous weeks, there are links between many pairs of states. A unified strategy will be crucial to stopping this outbreak.

<!-- This is no right-side text -->


<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text 13-->
# [Two local transmission chains circulating in Ontario](https://nextstrain.org/ncov/2020-04-02branchLabel=clade&f_division=Ontario&label=clade:B1)

Here, we see two clusters of sequences from Ontario. Each of these clusters is more closely related to sequences from other countries than to each other, indicating that these were the result of two separate introductions.
<br><br>
Within each cluster, however, each of these cases is most closely related to other cases from Ontario, indicating that this is most likely local, community transmission.
<br><br>
From this, we conclude that there has been community transmission in Ontario since at least the 3rd of March.

<!-- There is no right side text -->




<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text 13-->
# [What you can do](https://nextstrain.org/ncov/2020-03-27?c=country&d=map&p=full)
#### ...as an individual
* Practice strict social distancing, especially if you are in a vulnerable group.
* Remember that even if you are not super vulnerable, many people around you are; follow these practices to protect others.
* Wash your hands "like you just chopped a jalapeno and have to change a contact lens."  
* Stay home as much as possible -- especially  if you are sick; be prepared with extra supplies in case you need to self-quarantine.  
* If you are an employer, encourage your employees to work from home wherever possible.

#### ...as an official  
* Make testing free and broadly available.  
* Put strong social distancing measures in place.  
* Fund and implement extensive contact tracing efforts.  
* Financially support those impacted by social distancing measures.


<!-- This is the right-side text -->

```auspiceMainDisplayMarkdown
# Takeaways

#### We find evidence for local transmission within Senegal and the DRC. Swift action must be taken to contain the outbreak in these vulnerable areas.  
<br><br>
#### The origins of the Japanese outbreak may be associated with cruiseship travel  
<br><br>
#### Extensive sequencing by Iceland highlights extent of travel-associated spread and reveal likely local transmission in Austria  
<br><br>
#### Cases across the U.S. co-mingle on the tree, demonstrating extensive mixing across state lines  
```
<!-- ############ SLIDE BREAK ############# -->






<!-- ############ SLIDE BREAK ############# -->

<!-- This is left-side text -->
# [Scientific credit](https://nextstrain.org/ncov/2020-04-02?d=map&c=author)

We would like to acknowledge the amazing and timely work done by all scientists involved in this outbreak, and particularly those working in China.
Only through the rapid sharing of genomic data and metadata are analyses such as these possible.

<br>

We also gratefully acknowledge [GISAID](https://gisaid.org) for providing the platform through which these data can be uploaded and shared.

<!-- Do not need to translate institutions names -->
<!-- This is right-side text -->
```auspiceMainDisplayMarkdown

We are grateful for the data gathered by these originating labs:



```


<!-- ############ SLIDE BREAK ############# -->

<!-- This is left-side text -->
# [Detailed scientific credit](https://nextstrain.org/ncov/2020-03-05?d=map&c=author)

These data were shared via [GISAID](https://gisaid.org).
We gratefully acknowledge their contributions.

<br>

To the right we give specific sequences shared by each lab.

<!-- This is right-side text -->
```auspiceMainDisplayMarkdown

The SARS-CoV-2 genomes were generously shared by scientists at these submitting labs:




```
